GasanaJr/Capstone-backend
# User Messages Handling API

#### What does this PR do?
This PR adds the functionality of handling user queries/ messages in the database using Node Js, Express, and MongoDB
#### Description of Task to be completed?
Have the following endpoints working
GET      '/ '  : For retrieving all messages sent by a user
GET     '/:msgId' : For retrieving a certain message
POST  '/send' : For sending a message
#### How should this be manually tested?
Clone and run the files
Use Postman to test the endpoints mentioned above using this header
Key: Content-Type Value: application/json
Note that you have to be authenticated to do this. Get your JWT token for this by logging in
:
